backstage/src/loader/init.ts

```diff
@@ -2,8 +2,28 @@ import { GameContext } from '@app/executor/game_context'
 import { CompiledRuleSet } from '@app/loader/index'
 import { triggerEvent } from '@app/executor/events'
 import { applyStartup } from '@app/executor/startup'
-import { computeModifier, computePotentialAscensionPerks, computePotentialSkills } from '@app/executor/compute'
+import {
+   computeModifier,
+   computePotentialAscensionPerks,
+   computePotentialSkills
+} from '@app/executor/compute'
 import { initMap } from '@app/executor/map_site'
+import { initProperty } from '@app/executor/property'
+
+function setupCommonProperties(gameContext: GameContext) {
```

Comment:
perhaps we can make these into a startup or rule set initialization event?

Reply:
Then we'll need to stablize initialization order. Leave that to further pull requests.